docs/get-started/nextjs.md

```diff
@@ -0,0 +1,921 @@
+
+Note: Not all webpack modifications are copy/paste-able between `next.config.js` and `.storybook/main.js|ts`. It is recommended to do your research on how to properly make your modification to Storybook's webpack config and on how [webpack works](https://webpack.js.org/concepts/).
+
+Below is an example of how to add svgr support to Storybook with this framework.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/nextjs-configure-svgr.js.mdx',
+    'react/nextjs-configure-svgr.ts.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Typescript
+
+Storybook handles most [Typescript](https://www.typescriptlang.org/) configurations, but this framework adds additional support for Next.js's support for [Absolute Imports and Module path aliases](https://nextjs.org/docs/pages/building-your-application/configuring/absolute-imports-and-module-aliases). In short, it takes into account your `tsconfig.json`'s [baseUrl](https://www.typescriptlang.org/tsconfig#baseUrl) and [paths](https://www.typescriptlang.org/tsconfig#paths). Thus, a `tsconfig.json` like the one below would work out of the box.
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": ["components/*"]
+    }
+  }
+}
+```
+
+## React Server Components (RSC)
+
+(⚠️ **Experimental**)
+
+If your app uses [React Server Components (RSC)](https://nextjs.org/docs/app/building-your-application/rendering/server-components), Storybook can render them in stories in the browser.
+
+To enable this set the `experimentalRSC` feature flag in your `.storybook/main.js|ts` config:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/rsc-feature-flag.js.mdx',
+    'react/rsc-feature-flag.ts.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+Setting this flag automatically wraps your story in a [Suspense](https://react.dev/reference/react/Suspense) wrapper, which is able to render asynchronous components in NextJS's version of React.
+
+If this wrapper causes problems in any of your existing stories, you can selectively disable it using the `react.rsc` [parameter](https://storybook.js.org/docs/writing-stories/parameters) at the global/component/story level:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/rsc-parameter-in-meta.js.mdx',
+    'react/rsc-parameter-in-meta.ts.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+Note that wrapping your server components in Suspense does not help if your server components access server-side resources like the file system or Node-specific libraries. To deal work around this, you'll need to mock out your data access layer using [Webpack aliases](https://webpack.js.org/configuration/resolve/#resolvealias) or an addon like [storybook-addon-module-mock](https://storybook.js.org/addons/storybook-addon-module-mock).
```

Comment:
Correct me if I'm wrong, but am I missing something here, or is this "To deal work around" a typo? Shouldn't this be something like "To deal with...", "To workaround this", or something of that nature?